### DISCUSSION TITLE
Arturia Synths endless loop

### DISCUSSION THREAD

**Discussion by awedel68** [apohl79/audiogridder#1106] [Help]
Hello Andreas,
I am having an issue when trying to load Arturia instruments via Audiogridder in Reaper.
Plugin connects to server and after selecting VST or VST3 instrument the audiogridder server seems to enter an enternal loop.
This happens after trying to load the GUI, but I am not sure. I have to kill instances via Taskmanager.
The Plugins work, when loading them directly into reaper. I tried with audiogridder plugin on the same machine as the server.
Same happens using the plugin on a remote machine. I already tried rescan, re-install etc. I attached server and plugin logs and 
would appreciate any help. OP is Windows 7 64Bit. plugin and Server, latest beta 15.
[AudioGridder.zip](https://github.com/apohl79/audiogridder/files/10141804/AudioGridder.zip)
Best regards, Andreas

**Comment by awedel68** [comment]
![Screenshot](https://user-images.githubusercontent.com/119621100/205314989-5c752133-2a73-4b63-b4be-1f48f7717aaf.jpg)
I attached a screensot on the remote machine. there you can see, that the entire desktop of the server is being transmitted,
but the server seems to fail on loading the plugin. The top of the Plugin Window appears, sometimes the whole, but
then closes and another server instance is started.

**Comment by awedel68** [comment]
![Screenshot](https://user-images.githubusercontent.com/119621100/205316464-5c2c553b-de89-4c54-97bb-5b3ed7d9fb41.jpg)
Sorry for not posting it into one, but  other plugins (Effects and Synths) work.
![Screenshot2](https://user-images.githubusercontent.com/119621100/205316605-6e642bc4-e358-4582-b386-8f466667b4a3.jpg)

**Comment by apohl79** [comment]
It might be a compatibility issue. Can you check the server log and see if the server is able to load it? https://audiogridder.com/bug-report-diagnostics-locations/
